homework/index.js

```diff
@@ -30,15 +32,138 @@
+        createAndAppend('hr', div);
+
+        const table = createAndAppend('ul', div, { class: 'grid' });
+        const contributors = response;
+        contributors.forEach((_contributor, index) => {
+          const imageUrl = contributors[index].avatar_url;
+          const name = contributors[index].login;
+          const contributionsCount = contributors[index].contributions;
+          const profileURL = contributors[index].html_url;
+          const tr = createAndAppend('li', table, { class: 'clickable row' });
+          tr.onclick = () => openNewTab(profileURL);
+          const td1 = createAndAppend('div', tr, { class: 'col-30' });
+          const imgContainerDiv = createAndAppend('div', td1, {
+            class: 'image-container',
+          });
+          createAndAppend('img', imgContainerDiv, {
+            src: imageUrl,
+            width: 80,
+            height: 80,
+          });
+          createAndAppend('div', tr, {
+            text: name,
+            class: 'col-30 description',
+          });
+          const td3 = createAndAppend('div', tr, { class: 'col-30' });
+          createAndAppend('label', td3, { text: contributionsCount });
+        });
+      })
+      .catch(error => {
+        createAndAppend('div', contributorsDiv, {
+          text: error.message,
+          class: 'alert-error',
+        });
+      });
+  }
+  function bindRepositoryDetails(selectedRepository) {
+    const { description, forks } = selectedRepository;
+    const repositoryName = selectedRepository.name;
+    const updatedOn = new Date(selectedRepository.updated_at).toLocaleString();
+    const repoURL = selectedRepository.html_url;
+
+    const infoContainerDiv = document.getElementById('repo-info-container');
+    const repoInfoDiv = createAndAppend('div', infoContainerDiv, {
+      class: 'repo-info',
+    });
+    const div = createAndAppend('div', repoInfoDiv);
+    const table = createAndAppend('div', div, { class: 'grid' });
+
+    const tr1 = createAndAppend('div', table, { class: 'row' });
+    createAndAppend('div', tr1, { text: 'Repository:', class: 'col-50' });
+    const repoNameCol = createAndAppend('div', tr1, { class: 'col-50' });
+    createAndAppend('a', repoNameCol, {
+      href: repoURL,
+      target: '_blank',
+      text: repositoryName,
+    });
+
+    const tr2 = createAndAppend('div', table, { class: 'row' });
+    createAndAppend('div', tr2, { text: 'Description:', class: 'col-50' });
+    createAndAppend('div', tr2, { text: description, class: 'col-50' });
+
+    const tr3 = createAndAppend('div', table, { class: 'row' });
+    createAndAppend('div', tr3, { text: 'Forks:', class: 'col-50' });
+    createAndAppend('div', tr3, { text: forks, class: 'col-50' });
+
+    const tr4 = createAndAppend('div', table, { class: 'row' });
+    createAndAppend('div', tr4, { text: 'Updated:', class: 'col-50' });
+    createAndAppend('div', tr4, { text: updatedOn, class: 'col-50' });
```

Comment:
You are repeating yourself here four times for each 'row'. Can you think of a function that you could write to make your code DRY?